test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -986,7 +988,7 @@ def test_invalid_field
NIunRxh/NPqMwIhNNZumjQ219mWMceZE8O9g32DIwdLPbAtqEySvm3SOE1hgp9lHZC/efh7bdPRBTXLngrYUXk4tmT5ml6mO9vp4zMl7KPTNwVYfAUPMsY1li7fI0evDFca2QRXumRzVWZxUQVu7pjYptjUeWYCp5gXmW5Z7LIeqtpd7BXioD0pxHb0MVzprXcZh8eaupE5hw5euH5pyNM10o2yCik7Tlk45ntbCEbhu4O91NGTr+2FFt76B3d2VOh4qlQhsHoMyCSmsHFSv0OTLTuKouQo0Mb9np6C9YeXorXVUXwKw7NZjk2wz+6gwqMBcGcEAgrDkLJoDCqxP+xlEdKam2SWls3vGAx1An4jP1EPvrVLwqfusMiulUMaj5squ6InADda6xmm0prUKc4F5B2wY4wIvWgVdGByKm/sLEeHcw1/pRWCdMXdWu4gvXWj1TJvPN7cQx5P5kyoMM/oZBRxgI39/aLFOHM7k+UEkQ2phclmKlF0KYaieMWQbrY3T8QLWPUoJW5PEY1xIlmo0OJWYIqTeFIu3UdRJKBw/cPphOq3L25zilNle4vMSNfbMss8dU5cu6lWLQc8cL6d57cxWegbFcMVskbXTyM9UzVm8eRbNxuJ6i9i6OaN8otBlwrRsGj7IMQjmeAP0pejIj0eGtEck3SR3murmkaLQpiCtw2rKboMub2DN5By7yuYmjJTJ5nUpXnln8k6Xp2BjIqfqbpw9Nck2Nhx3hIkCqyok+bLVRb0QIPyTFosGfFrZOgrbF6bK9AG94UHDDljA5dU1u2GSKQnnrbq5YvkKC7FTWt5KuuaSSJ8rVyCoARDgCfJ/Rs+fwSmXmjDJ8es3OI205Ql3Tug9qzwkW8vOSGgn9qdwSvyv4FT8UhRecMr9JZxeqXvEfIPSSZkBrhTWH6H0tfcDlO7H/wKnxduPkMhNfxMQDVgO09eZAzdFpmerMK3Da4nGBp0FVy19ShsKUwM1LDyTuyiM+LIBmJTgL+xmwv+4b6sQcOB99D+BzY7NgtGkA2/oqTidFXG90bd+DWFiUvoQix/WCWMwM57ON7/ppYmklVlaW0paZOGhzBlemVR+qgkevfU9HdeaJAvlhjPtzUVcBbp6ERdvwUWZlXJSxkRMkLQ47KeL2AXx0rcPDHj49ZbCtak6Iy1j2KdJc/OsnFuMO0/JJshDfFDcKZQACksyKGkhEpKeOhgEWmj2gLvHzqSwmG6EJHcPfe7JnNL5Q5yRTaJ7ZxdEqpta6+NmKI9RsXZW2BTJMIWco5lKpwxlDOFqskgCW/feqaqbwHPxIVUBBa5HGxu1s9gRS3xJilz1SGvVW6n3+LI+YRU5lhjpw+I+9+dsDwaSGPvBCR+HLTrwN0HkuMQ85D/AJmAZGuK08QFEygsgGRYIT/AE6bOBUcD6A/wgPCAjdA7MPbqg26/Q5bF0rghM6tzoMCubvwddd+/qwQwhShfn7j+NH+uPEfojDfTdV6qzdA8goGkGUYTXcZTTF6TrNKOMoq/BsohOwgy8r/nHlR6M/9L07Wgwz/Rh8WYY6ONgaqjKgg8w1tYcD1vL8MQPQ2s7dJXt9f0asJu5YEpk4N31uB9ftxQ0nY58DawN3cFCigaRXxRYfCWw+A6TYy3Ya369fjaaklx7YnYPVRjbB1oDbJrCLkQ2R5c0Bvze0IpSduUU4LZ2khZvmRXFlcitmocsaAhQXVKEFtaz4/Y+gDNG+xtFEJv9Uldhospmmuenop4qwVweCFNW1Jub3ZbXDsa4GJT7IBCqiTkP0UqasTChcnOzje/BPExDFd2vtXSdV5ywmamtMWgtoQcZY3Vrcpusr2PTDylxb6DvltoUFKHBW1vWu5/y++WQ0I+KkpI0doVamNhyv99LOXiUqWoS8p5z2usQ2lq8N1ztkBwnI2EIl+BhBkOFLrOblyM+BVdu3crexLjjFFLx7F2vp+S6tpepJCvA5iLxeCsyX2BRjcLujnOIz6HjFNVcx8ribRu3mKw/SLI+0p3U0Lds86AthvMxLmwD8W+CCrspIKj08jdQOfhsgY+EvdpXaciruV+t5Hz+fwoq83+DyuV/BSrZl/7xCSruX9jNCHAKfcIJlOkfAMqDNsdrZ0qYtU+kfRO3Kr3fLOf7IJMBtPh8FwAQxaS469pD4sSYkMExzkwRsI8pxSfLvfE2ZoJbu1wTSEN7h9MJ72LYBHr9LLaXBklAWcMZLcSdW2nZ43K8HwYh5uW+P/cGW4mISDQXWkEvyWmPhSE4FJvYpHqKH7fnq3cmEi+8DYfK2neyoJq3xRsGgprr4STuNewMchIluBlxlbzaIkKIyXJSHnlkRZoW1UxRerIqlLKvQxBHoSj2BrHNLmYrpQGEArNJWt0R7oQaDMvgkbAy2JuXOlIYWwYjaytKP0mPaBXMrEneOKxk7NmaSvqiauzmqFty4tQhHkEYx06YwWl9MttlglPSCemYm4W6Wf0VUPD1GLB/CSb0VyiBM9ofwOQ5b3xMTZBP3NDjKBQcnCDcr9FwehbrZ6lWWG19MLleMRXsOZVB8P02l5m1MLZ6j92Op/MmqM4bEgZf6gWpUxQ/7+1frcY54IprBFuLGtdPW9vbt4V1K8+q16krBA3p6TLWbNQhy4RSUnnAHxzZbFy8a+j5iKs+mUeSxDrgmJ+2DXZDwjPUjh3dVWscZWNjatQNSzEMIp0P9PkSmY0WDAYb33yrOabUwMFZAZNKd+ba8CJ0zFKZG3GbBFxzvy/ecI2L49IrbHlQL3CmlDbMXImRfGCXV3GFngZtLubb/Xy7SkNR+14iy0KyqntWbIOLSdaNgVsscYPxpJ9aTnFWlerP+TlSBL/Abl5Jmc76dFGH5sqmTHdxHtC8FgKnLCp3nESZD3T+uIsOTnR6n6A9IRwXb5VwJpb2GW277ORjamXx7Uid65hrBZSYMZK63UiDyq/2aJJr7ae9PfL9zR3y7W3e9/d8r98zXj+4PF/B//hDzH8AlPBTsQ==
```

Comment:
🎉

Reply:
The one thing I'd like to at least capture in the commit message is the links and overall steps to obtain this value, perhaps?

Reply:
> The one thing I'd like to at least capture in the commit message is the links and overall steps to obtain this value, perhaps?

@esmitperez added the link and instructions in a comment above the tests to make it obvious 😄